<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>715</width>
                <height>588</height>
            </rect>
        </property>
        <property name="font">
            <font>
                <family>Times</family>
                <pointsize>15</pointsize>
                <weight>9</weight>
                <italic>true</italic>
                <bold>false</bold>
            </font>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="styleSheet">
            <string notr="true">
                font: 75 italic 15pt &quot;Times&quot;;
            </string>
        </property>
        <property name="animated">
            <bool>true</bool>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QDateTimeEdit" name="dateTimeEdit">
                <property name="geometry">
                    <rect>
                        <x>220</x>
                        <y>420</y>
                        <width>201</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="currentSectionIndex">
                    <number>0</number>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
                <property name="geometry">
                    <rect>
                        <x>140</x>
                        <y>500</y>
                        <width>131</width>
                        <height>71</height>
                    </rect>
                </property>
                <property name="text">
                    <string>DATE FORMAT</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
                <property name="geometry">
                    <rect>
                        <x>360</x>
                        <y>500</y>
                        <width>121</width>
                        <height>71</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">font: 75 italic 14pt &quot;Times&quot;;</string>
                </property>
                <property name="text">
                    <string>TIME FORMAT</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>260</y>
                        <width>131</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="text">
                    <string>STOP!</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_4">
                <property name="geometry">
                    <rect>
                        <x>170</x>
                        <y>70</y>
                        <width>121</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>RESET!</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_5">
                <property name="geometry">
                    <rect>
                        <x>400</x>
                        <y>70</y>
                        <width>111</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>STOP!</string>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber">
                <property name="geometry">
                    <rect>
                        <x>200</x>
                        <y>200</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>2</number>
                </property>
                <property name="value" stdset="0">
                    <double>0.000000000000000</double>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber_2">
                <property name="geometry">
                    <rect>
                        <x>320</x>
                        <y>200</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>2</number>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber_3">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>200</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>2</number>
                </property>
            </widget>
            <widget class="QSpinBox" name="spinBox">
                <property name="geometry">
                    <rect>
                        <x>60</x>
                        <y>220</y>
                        <width>42</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QSpinBox" name="spinBox_2">
                <property name="geometry">
                    <rect>
                        <x>60</x>
                        <y>250</y>
                        <width>42</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QSpinBox" name="spinBox_3">
                <property name="geometry">
                    <rect>
                        <x>60</x>
                        <y>280</y>
                        <width>42</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber_4">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>20</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>2</number>
                </property>
                <property name="value" stdset="0">
                    <double>0.000000000000000</double>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber_5">
                <property name="geometry">
                    <rect>
                        <x>250</x>
                        <y>20</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>2</number>
                </property>
                <property name="value" stdset="0">
                    <double>0.000000000000000</double>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber_6">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>20</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>2</number>
                </property>
                <property name="value" stdset="0">
                    <double>0.000000000000000</double>
                </property>
            </widget>
            <widget class="QLCDNumber" name="lcdNumber_7">
                <property name="geometry">
                    <rect>
                        <x>490</x>
                        <y>20</y>
                        <width>81</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="digitCount">
                    <number>3</number>
                </property>
                <property name="value" stdset="0">
                    <double>0.000000000000000</double>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_6">
                <property name="geometry">
                    <rect>
                        <x>200</x>
                        <y>260</y>
                        <width>131</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="text">
                    <string>RESET!</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_7">
                <property name="geometry">
                    <rect>
                        <x>290</x>
                        <y>340</y>
                        <width>131</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="text">
                    <string>START!</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_8">
                <property name="geometry">
                    <rect>
                        <x>280</x>
                        <y>120</y>
                        <width>131</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>START!</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
